controllers/gatekeeper_controller_test.go

```diff
@@ -48,6 +48,53 @@ func TestDeployValidatingWebhookConfig(t *testing.T) {
 	g.Expect(getStaticAssets(gatekeeper)).NotTo(ContainElement(ValidatingWebhookConfiguration))
 }
 
+func TestCustomNamespace(t *testing.T) {
```

Comment:
This checks the corner cases and I think that's probably good enough as the other manifests are verified in the e2e tests.